### PR TITLE
Add documentation for `bento-fit-text` and `BentoFitText`

### PR DESCRIPTION
**Pull request by caroqliu**
The new `README.md` is meant for npm publishing and is distinct from `amp-fit-text.md`, which is specific to AMP documents.

### CODE REVIEW

File: extensions/amp-fit-text/1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+If the content of the `bento-fit-text` overflows the available space, even with a
+`min-font-size` specified, the overflowing content is cut off and hidden. WebKit and Blink-based browsers show ellipsis for overflowing content.
+
+In the following example, we specified a `min-font-size` of `40`, and added more content inside the `bento-fit-text` element. This causes the content to exceed the size of its fixed block parent, so the text is truncated to fit the container.
+
+[example preview="inline" playground="true" imports="bento-fit-text:1.0"]
+
+```html
+<div style="width: 300px; height: 300px; background:#005AF0; color: #FFF;">
```
Comment (alanorozco) [inline extensions/amp-fit-text/1.0/README.md:89]: nit
```suggestion
<div style="width: 300px; height: 300px; background: #005AF0; color: #FFF;">
```

File: extensions/amp-fit-text/1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+[/example]
+
+#### Layout and style
+
+**Container type**
+
+The `BentoFitText` component has a defined layout size type. To ensure the component renders correctly, be sure to apply a size to the component and its immediate children (slides) via a desired CSS layout (such as one defined with `height`, `width`, `aspect-ratio`, or other such properties). These can be applied inline:
+
+```jsx
+<BentoFitText style={{width: '300px', height: '100px'}}>
```
Comment (alanorozco) [inline extensions/amp-fit-text/1.0/README.md:188]: Ok to leave as is but consider removing 'px' and making these numbers. They'll be rendered as pixel values by default.

```suggestion
<BentoFitText style={{width: 300, height: 100}}>
```

**Review by alanorozco — APPROVED** [review]

File: extensions/amp-fit-text/1.0/README.md (hunk 2 above)
Comment (caroqliu) [reply extensions/amp-fit-text/1.0/README.md:188]: Looks a lot cleaner, thanks!

File: extensions/amp-fit-text/1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+#### Attributes
+
+##### Media Queries
+
+The attributes for `<bento-fit-text>` can be configured to use different
+options based on a [media query](./../../../docs/spec/amp-html-responsive-attributes.md).
+
+##### `min-font-size`
+
+Specifies the minimum font size as an integer that the `bento-fit-text` can use.
```
Comment (kvchari) [inline extensions/amp-fit-text/1.0/README.md:144]: I'm assuming this is in pixels, but is it worth calling out the units here?

**Review by kvchari — APPROVED** [review]

File: extensions/amp-fit-text/1.0/README.md (hunk 3 above)
Comment (caroqliu) [reply extensions/amp-fit-text/1.0/README.md:144]: Good idea, updated in all locations.

**Pull request merged**